### DISCUSSION TITLE
不能收藏音乐

### DISCUSSION THREAD

**Discussion by Amoelcold** [qier222/YesPlayMusic#1815] [Q&A]
只能点红星把音乐收藏到喜欢的音乐歌单了，但不能把音乐添加到其他歌单。
![GIF 2022-10-3 19-26-03](https://user-images.githubusercontent.com/55040556/193565975-e9f4232f-9f5b-427f-9cdd-5623abb13dd1.gif)

**Comment by GrassBlock1** [comment]
你点一下那个歌单卡片的背景就好了...
